metadata/sample.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Extra file here? We already have a sample release metadata in `release/sample.json`, right?

Reply:
A leftover indeed, good catch!